specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2023-01-01-preview/AlertRuleRecommendations.json

```diff
@@ -0,0 +1,251 @@
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/{resourceUri}/providers/Microsoft.AlertsManagement/alertRuleRecommendations": {
+      "get": {
+        "tags": [
+          "alertRuleRecommendations"
+        ],
+        "description": "Retrieve alert rule recommendations for a resource.",
+        "operationId": "AlertRuleRecommendations_ListByResource",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceUriParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request for a list of alert rule recommendations",
+            "schema": {
+              "$ref": "#/definitions/AlertRuleRecommendationsListResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List alert rule recommendations for virtual machines at resource level": {
+            "$ref": "./examples/AlertRuleRecommendations_GetByResource_VM.json"
+          },
+          "List alert rule recommendations for Monitoring accounts at resource level": {
+            "$ref": "./examples/AlertRuleRecommendations_GetByResource_MAC.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AlertsManagement/alertRuleRecommendations": {
+      "get": {
+        "tags": [
+          "alertRuleRecommendations"
+        ],
+        "description": "Retrieve alert rule recommendations for a target type.",
+        "operationId": "AlertRuleRecommendations_ListByTargetType",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/TargetTypeParameter"
+          }
+        ],
+        "responses": {
```

Comment:
>"responses" [](http://example.com/codeflow?start=8&length=11)

There is a Linter issue Warning:

Based on the response model schema, operation 'AlertRuleRecommendations_ListByResource' might be pageable. Consider adding the x-ms-pageable extension.